cfpb/cfgov-refresh
# Automatically populate the latest updates on the Home Page

This PR replaces the manually-entered list of latest updates on the homepage with an automatically populated list. Right now it's behind a new feature flag, `AUTOMATIC_LATEST_UPDATES`. 

## Testing

1. Visit http://localhost:8000/ and note the manually entered Latest Updates section
2. Visit http://localhost:8000/admin/flags/ and add a `boolean` `True` condition for `AUTOMATIC_LATEST_UPDATES`
3. Visit http://localhost:8000/ and note the automatically generated Latest Updates section
4. Edit the homepage and add an excluded page
5. Visit http://localhost:8000/ and note the page you excluded is no longer in the generated Latest Updates section.

## Screenshots

![image](https://user-images.githubusercontent.com/10562538/54753442-737a3d00-4bb7-11e9-9780-48e842cfa0e8.png)

![image](https://user-images.githubusercontent.com/10562538/54753465-7d9c3b80-4bb7-11e9-8835-f34a9d246beb.png)

## Checklist
